onnxruntime/test/fuzzing/src/OnnxPrediction.cpp

```diff
@@ -45,101 +62,83 @@ session{nullptr}
 
```

Comment:
It's confusing to be using the one OnnxPrediction class for both ONNX and ORT format models. Can we lift the common functionality into a Prediction class, and derive or compose (prefer compose if possible) an OnnxPrediction and OrtFormatPrediction class from that? An 'OnnxPrediction' ctor that takes a buffer and internally forces that to be treated as ORT could be unexpected.

Reply:
The class has two constructors one used by the ONNX flow (ModelProto) and the other use by ORT flow (flatbuffer passed asvector<char>). Other than that we are not doing any thing specific. The constructors create the Session object in both cases using the model and the other functionality works with the session object. I am not sure in this case we require a class hierarchy. We could take this up later if we add code specific to ORT or ONNX.